### PR TITLE
Add normalizers

### PR DESCRIPTION
**Pull request by shavit**
• Lowercase
• NFD
• NFC
• NFKD
• NFKC

Related #4

### CODE REVIEW

**Comment by pcuenca** [comment]
Nice! Let me know when you want a review :)

**Comment by shavit** [comment]
Yes please, I will also give another look at the tests.

**Review by pcuenca — COMMENTED** [review]
This looks great so far! Is there anything else you're planning to add, or are you basically done?

File: Tests/NormalizerTests/NormalizerTests.swift [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    }
+    
+    func testBertNormalizer() {
+        let testCases: [(String, String)] = [
+            ("Café", "café"),
+            ("François", "françois"),
+            ("Ωmega", "ωmega"),
+            ("über", "über"),
+            ("háček", "háček"),
+            ("Häagen\tDazs", "häagen dazs"),
```
[the remark this answers is not in the document]
Comment (shavit) [reply Tests/NormalizerTests/NormalizerTests.swift:114]: Yes, I only planned to improve the tests.

File: Tests/NormalizerTests/NormalizerTests.swift [hunk 2]
```diff
@@ -0,0 +1,165 @@
+import XCTest
+@testable import Tokenizers
+@testable import Hub
+
+class NormalizerTests: XCTestCase {
```
[the remark this answers is not in the document]
Comment (shavit) [reply Tests/NormalizerTests/NormalizerTests.swift:5]: Yes, except the precompiled normalizer.

[the remark this answers is not in the document]
Comment (pcuenca) [reply Tests/NormalizerTests/NormalizerTests.swift:5]: Awesome! Do you need help with it? I was going to write it anyway for #38 :)

[the remark this answers is not in the document]
Comment (shavit) [reply Tests/NormalizerTests/NormalizerTests.swift:5]: Yes.

[the remark this answers is not in the document]
Comment (pcuenca) [reply Tests/NormalizerTests/NormalizerTests.swift:5]: Ok, I'll take a look tomorrow.

**Review by pcuenca — COMMENTED** [review]
I tested the `Precompiled` normalizer against the Python and the JavaScript version, and added a couple of additional test cases. The current implementation is not complete, but it appears to be identical to the one in `transformers.js` and is good enough for now, in my opinion.

I also added a couple of comments as pointers for future work.

Could you please incorporate these suggestions, and then we can merge? Thank you! 🙌

**Review by pcuenca — APPROVED** [review]
Thanks a lot @shavit!

**Pull request merged**